### PR TITLE
feat: create request test by category page (dbio-308)

### PR DESCRIPTION
**Pull request by ciniiia**
### JIRA Link
https://blocksphere2020.atlassian.net/browse/DBIO-308?atlOrigin=eyJpIjoiNjNkNzRlN2I1ODI3NGFmMTliZDIxMGQ2ZGQ3NTVhM2QiLCJwIjoiaiJ9

### Changelog / Description
- create page request test by category

### CODE REVIEW

**Comment by bukandicki** [comment]
LGTM 👍🏻
ada beberapa hal kecil aja soal konsistensi penggunaan tanda petik

**Review by bukandicki — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Customer/RequestTest/FindTest.vue [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+    await this.getCountries();
+  },
+  computed: {
+    ...mapState({
+      api: (state) => state.substrate.api,
+      wallet: (state) => state.substrate.wallet,
+      configApp: (state) => state.auth.configApp,
+    }),
+    citiesSelection() {
+      return this.cities
```
[the remark this answers is not in the document]
Comment (ciniiia) [reply src/views/Dashboard/Customer/RequestTest/FindTest.vue]: mas @bukandicki, minta tolong di reviewin lagi yaa. thank you

**Review by bukandicki — APPROVED** [review]

File: src/views/Dashboard/Customer/RequestTest/FindTest.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (bukandicki) [reply src/views/Dashboard/Customer/RequestTest/FindTest.vue]: done mbak @ciniiia

**Review by kevinjanada — APPROVED** [review]

**Comment by kevinjanada** [comment]
Akan dibuat PR baru karena ada perubahan requirements

**Pull request closed without merging**